### PR TITLE
Improve wording of selective checks comments

### PR DESCRIPTION
**Pull request by potiuk**
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

File: .github/workflows/label_when_reviewed_workflow_run.yml [hunk 1]
```diff
@@ -97,21 +97,23 @@ jobs:
           require_committers_approval: 'true'
           pullRequestNumber: ${{ steps.source-run-info.outputs.pullRequestNumber }}
           comment: >
-            The PR needs to run all tests because it modifies core of Airflow! Please rebase it to latest
-            master or ask committer to re-run it!
+            The PR most likely needs to run full matrix of tests because it modifies parts of the core
+            of Airflow. However, committers might decide to merge it quickly and take the risk.
+            If they don't merge it quickly - please rebase it to latest master at your convenience,
+            or amend the last commit of the PR, and push it with --force.
```
Comment (ashb) [inline .github/workflows/label_when_reviewed_workflow_run.yml]:
```suggestion
            or amend the last commit of the PR, and push it with `--force-with-lease`.
```

Safer than `--force`

Comment (potiuk) [reply .github/workflows/label_when_reviewed_workflow_run.yml]: Good point! Changed everyewhere.

File: .github/workflows/label_when_reviewed_workflow_run.yml [hunk 2]
```diff
@@ -97,21 +97,23 @@ jobs:
           require_committers_approval: 'true'
           pullRequestNumber: ${{ steps.source-run-info.outputs.pullRequestNumber }}
           comment: >
-            The PR needs to run all tests because it modifies core of Airflow! Please rebase it to latest
-            master or ask committer to re-run it!
+            The PR most likely needs to run full matrix of tests because it modifies parts of the core
+            of Airflow. However, committers might decide to merge it quickly and take the risk.
+            If they don't merge it quickly - please rebase it to latest master at your convenience,
```
Comment (XD-DENG) [inline .github/workflows/label_when_reviewed_workflow_run.yml]:
```suggestion
            If they don't merge it quickly - please rebase it to the latest master at your convenience,
```

nit.

File: .github/workflows/label_when_reviewed_workflow_run.yml [hunk 3]
```diff
@@ -97,21 +97,23 @@ jobs:
...
+            or amend the last commit of the PR, and push it with --force-with-lease.
       - name: "Initiate GitHub Check forcing rerun of SH ${{ github.event.pull_request.head.sha }}"
         uses: LouisBrunner/checks-action@9f02872da71b6f558c6a6f190f925dde5e4d8798  # v1.1.0
         id: full-test-check
         if: steps.label-full-test-prs-when-approved-by-commiters.outputs.labelSet == 'true'
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
-          name: "Please rebase or re-run to run full tests"
+          name: "Please rebase or amend and force push the PR to run full tests"
```
Comment (XD-DENG) [inline .github/workflows/label_when_reviewed_workflow_run.yml]:
```suggestion
          name: "Please rebase or amend, and force push the PR to run full tests"
```

**Review by XD-DENG — COMMENTED** [review]
nit

**Review by XD-DENG — APPROVED** [review]

**Review by turbaszek — APPROVED** [review]

**Pull request merged**